docs/tutorials/how-to-search-effective.md

```diff
@@ -0,0 +1,156 @@
+# How to search Amundsen effectively
+
+The goal of this tutorial is to provide a few tips on how to search for datasets effectively in Amundsen. 
+
+## Overview
+
+Amundsen currently indexes three types of entities: tables, people, and dashboards. This tutorial mostly covers how to search for a table entity effectively.
+
+We will cover other entities in the future.
+
+## General Search
+
+Once the users are on the Amundsen home page,  the users could search for any random information in the search bar. In the backend, the search system will use the same query term from users and search across three different entities (tables, people, and dashboards) and return the results with the highest ranking. For Table search, it will search across different fields, including table name, schema name, table or column descriptions, tags and etc. Amundsen also supports typeahead search which will search in the backend as soon as users enter new characters. 
+
+
+Tips:
+
+- If you know the full table name (e.g. schema.table), try to search with that full table name, which will provide as the top result in general.
+- If you are unsure of the table name, search with `word1 word2` with space in between. For example, if your table's name is `test.test_rides`  but you don't know the exact table name but only know the table name includes fact and rides, please search with `test rides` (space in between). In this case, Amundsen will search all the matching with fact  as well as rides and union the result together based on ranking.
```

Comment:
I think this might be a typo (or I'm misunderstanding) - should `fact` be replaced by `test`?
Also, just to clarify, when you search `test rides` you say it searches for matches with `fact as well as rides and union the result`. Can I confirm, is it looking for tables that contain fact AND rides, or fact OR rides?

Reply:
correct, initially the doc is for internal with an internal table name which I modify a bit before posting it in open source :) if you have time, maybe you could help to create a pull request for us